tests/e2e/ethereum_test.go

```diff
@@ -278,6 +281,16 @@ func (s *ethTestSuite) TestSignTransaction() {
 		httpError := err.(*client.ResponseError)
 		assert.Equal(s.T(), 404, httpError.StatusCode)
 	})
+
+	s.Run("should sign a big amount of transactions successfully", func() {
+		for i := 0; i < 500; i++ {
```

Comment:
What is the added value of this test unit? If we are confident about the fix we did there is not need for it. WDYT? 

In addition I would rather evaluate the long term cost of this before pushing

Reply:
removed